.evergreen/config.yml

```diff
@@ -994,9 +998,9 @@ buildvariants:
   - matrix_name: "lint"
     matrix_spec:
       lint: on
-      ruby: "ruby-2.6"
+      ruby: "ruby-2.7"
       mongodb-version: ["4.4"]
-      topology: ["standalone", "replica-set", "sharded-cluster"]
+      topology: '*'
       os: ubuntu1604
```

Comment:
Load balancer config is failing on ubuntu1604 because there is no haproxy there, let's try ubuntu1804 instead.